### PR TITLE
Fixes for Table viz

### LINKED ISSUE #7590: Sometimes table visualization columns don't auto-size correctly when visualization is initially opened
### Discord username

_No response_

### What type of issue is this?

Intermittent – Occurring irregularly

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

Sometimes when I initially open a Table visualization for a node, not all of the columns auto-size to fit their content:
image

If I drag a corner of the visualization to resize it, the columns resize to their content as soon as I have changed the visualization size at all.

### Expected behaviour

Columns auto-sizing should not change completely when a small visualization resize occurs; I think the intended behavior is for them always to be sized to their content.

### How we can reproduce it?

1. Open a table visualization.
2. Often, but not always the bug is apparent.

I think this has been occurring for a long time (on the scale of weeks).

### Screenshots or screencasts

_No response_

### Logs

_No response_

### Enso Version

develop (fd972f8)

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

standalone

### Operating System

Linux

### Operating System Version

_No response_

### Hardware you are using

_No response_

### LINKED ISSUE #8043: Table visualizations is showing an empty table if Date_Time column is included
As described in the title, here is a simple repro showing the problem:

image

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

- Fixes issue with `to_display_text` on `Date_Time`. Now a reversible format.
- Adjusted the auto width code on table so it works.
- Manually sized columns should remain fixed size.
- Enable Excel range style selection in table.
- Removed page support as not implemented yet.

### Important Notes

Will need to apply to Vue based viz at some point as well.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by farmaazon — APPROVED** [review]

File: test/Tests/src/Data/Time/Time_Of_Day_Spec.enso [hunk 1]
```diff
@@ -37,6 +37,10 @@ specWith name create_new_time parse_time nanoseconds_loss_in_precision=False =
             text = create_new_time 12 20 44 . format "HHmmss"
             text . should_equal "122044"
 
+        Test.specify "should support display_text" <|
+            text = create_new_time 12 20 44 . to_display_text
+            text . should_equal "12:20:44"
```
Comment (radeusgd) [inline test/Tests/src/Data/Time/Time_Of_Day_Spec.enso:42]: Let's also add a test case for when nanoseconds are displayed. 

```suggestion
            text = create_new_time 12 20 44 . to_display_text
            text . should_equal "12:20:44"
            
            create_new_time 12 20 44 123 . to_display_text . should_equal "12:20:44.000"
```

**Comment by radeusgd** [comment]
> * Enable Excel range style selection in table.

Could you please show us what this is? As I'm not exactly what this feature enables.

**Review by radeusgd — APPROVED** [review]
Thanks for fixing the formats.

I think ideally would be good to add one more test in `Time_Of_Day` to ensure we don't regress on this in the future.

**Comment by jdunkerley** [comment]
> > * Enable Excel range style selection in table.
> 
> Could you please show us what this is? As I'm not exactly what this feature enables.

Ability to select a block of cells:
image

**Review by GregoryTravis — APPROVED** [review]

**Pull request merged**